src/dioptra/restapi/v1/jobs/schema.py

```diff
@@ -41,14 +41,68 @@ class JobMlflowRunSchema(Schema):
     )
 
 
-class JobStatusSchema(Schema):
-    """The fields schema for the data in a Job status resource."""
-
+class JobIdSchema(Schema):
     id = fields.Integer(
         attribute="id",
         metadata=dict(description="ID for the Job resource."),
         dump_only=True,
     )
+
+
+class MetricsSchema(Schema):
+    name = fields.String(
+        attribute="name",
+        metadata=dict(description="The name of the metric."),
+        required=True,
+    )
+    value = fields.Float(
+        attribute="value",
+        metadata=dict(description="The value of the metric."),
+        required=True,
+    )
```

Comment:
let's add `step` as an `Int` defaulting to `0`.
This mimics mlflow client's `log_metric` function and we can pass it through there in the service layer. 
https://mlflow.org/docs/latest/python_api/mlflow.client.html#mlflow.client.MlflowClient.log_metric

Reply:
I added timestamp as well since that is an option in mlflow